src/vcpkg/commands.format-manifest.cpp

```diff
@@ -41,7 +41,8 @@ namespace
         auto res = serialize_manifest(*data.scf);
 
         // reparse res to ensure no semantic changes were made
-        auto maybe_reparsed = SourceControlFile::parse_project_manifest_object(StringView{}, res, null_sink);
+        auto maybe_reparsed =
```

Comment:
This is a drive-by fix because `format-manifest` crashes in debug builds because origin is empty.